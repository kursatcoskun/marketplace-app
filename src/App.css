.ant-pagination-item-active{
color: white;
background-color: red;
}