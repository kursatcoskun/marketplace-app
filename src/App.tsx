import React from "react";
import "./App.css";
import "antd/dist/antd.css";
import HeaderLayout from "./components/Header/index";
import HomeContainer from "./containers/HomeContainer";
import Footer from "./components/Footer";

function App() {
  return (
    <div className="app">
      <HeaderLayout title="Market" />
      <HomeContainer />
      <Footer>Â©2019 Market &#183; Privacy Policy</Footer>
    </div>
  );
}

export default App;
