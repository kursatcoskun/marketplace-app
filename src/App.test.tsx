// MyComponent.test.js
import { shallow } from "enzyme";
import App from "./App";
describe("App", () => {
  it("should render App", () => {
    const wrapper = shallow(<App />);
  });

  it("should render initial layout", () => {
    const component = shallow(<App />);
    expect(component.getElements()).toMatchSnapshot();
  });
});
