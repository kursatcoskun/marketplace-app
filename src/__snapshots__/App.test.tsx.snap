// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render initial layout 1`] = `
Array [
  <div
    className="app"
  >
    <Memo(Connect(HeaderLayout))
      title="Market"
    />
    <Memo(Connect(HomeContainer)) />
    <styled.div>
      ©2019 Market · Privacy Policy
    </styled.div>
  </div>,
]
`;
